.github/workflows/release.yml

```diff
@@ -99,6 +99,52 @@ jobs:
         name: ${{ steps.artifact-name.outputs.name }}
         path: target/${{ matrix.target }}/release/astar-collator
 
+  evm-tracing-native-linux:
+    needs: checks-and-tests
+    runs-on: [self-hosted, Linux, X64]
+    strategy:
+      matrix:
+        chain: ["astar", "shiden", "shibuya"]
+    steps:
+
+    - name: Checkout the source code
+      uses: actions/checkout@v3
+      with:
+        submodules: true
+
+    - name: Install deps
+      run: sudo apt -y install protobuf-compiler
+
+    - name: x86_64 setup
+      run: |
+        mkdir -p .cargo
+        touch .cargo/config
+        printf '[target.x86_64-unknown-linux-gnu]\nrustflags = ["-Clink-arg=-fuse-ld=lld"]' >> .cargo/config
+
+    - name: Install & display rust toolchain
+      run: rustup show
+
+    - name: Check targets are installed correctly
+      run: rustup target list --installed
+
+    - name: Build optimized binary with evm tracing
+      run: CARGO_PROFILE_RELEASE_LTO=true RUSTFLAGS="-C codegen-units=1" cargo build --release --target x86_64-unknown-linux-gnu --features evm-tracing --verbose --locked
+
+    - uses: actions/upload-artifact@v3
+      with:
+        name: astar-ubuntu-tracing-latest
+        path: target/x86_64-unknown-linux-gnu/release/astar-collator
+
+    - name: rename tracing runtimes
+      run: |
+        cd target/x86_64-unknown-linux-gnu/wbuild/${{ matrix.chain }}-runtime
+        mv ${{ matrix.chain }}_runtime.compact.compressed.wasm ${{ matrix.chain }}_tracing_runtime.compact.compressed.wasm
+
+    - uses: actions/upload-artifact@v3
+      with:
+        name: ${{ matrix.chain }}-tracing-runtime
+        path: target/x86_64-unknown-linux-gnu/wbuild/${{ matrix.chain }}-runtime/${{ matrix.chain }}_tracing_runtime.compact.compressed.wasm
```

Comment:
Even though directory is changed at the former step (`cd ...`), path to compressed.wasm can be resolved?
I'm asking because I'm not sure. You can test release CI by adding tag like `v0.0.0-tracing-test`.

Reply:
@shunsukew as far as I could see [here](https://github.com/AstarNetwork/Astar/blob/3d09c3a22fdce6cbe1b88bfbc4ee9c6604645449/.github/workflows/release.yml#L432) the change of relative path during a run action doesn't affect the path in next upload action [here](https://github.com/AstarNetwork/Astar/blob/3d09c3a22fdce6cbe1b88bfbc4ee9c6604645449/.github/workflows/release.yml#L448).

Reply:
I see, in that case no need to care about it. thank you